pkg/capabilities/capabilities.go

```diff
@@ -75,6 +75,13 @@ type CapabilityRequest struct {
 	Inputs   *values.Map
 }
 
+type TriggerEvent struct {
+	TriggerType    string
+	ID             string
+	Timestamp      string
+	BatchedPayload map[string]any
```

Comment:
Do we consider these always to be batched payload? Would it be more useful to consider this just a `payload` of type `map[string]any`? This would make it more generic, so that triggers can either accept batched payloads or single ones.

Reply:
Yeah we can call it payload and have a default constant "batch" name (or rather "shard" name?) for single-item payloads. I can do that separately since Cedric merged this one already.